airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -264,6 +276,21 @@ def _start_task(self):
         self.log.info('ECS Task started: %s', response)
 
         self.arn = response['tasks'][0]['taskArn']
+        ecs_task_id = self.arn.split("/")[-1]
+        self.log.info(f"ECS task ID is: {ecs_task_id}")
+
+        if self.reattach:
+            # Save the task ARN in XCom to be able to reattach it if needed
+            self._xcom_set(context, key="ecs_task_arn", value=self.arn, task_id=f"{self.task_id}_task_arn")
+
+    def _xcom_set(self, context, key, value, task_id):
```

Comment:
This just contains a single line and is only used once, do we need the helper? Or did you use this mostly to make patching in your testing easier (which is completely valid IMHO)?

Reply:
yes exactly I added it for the patching in the testing